src/harness/sas_interface.py

```diff
@@ -287,4 +287,36 @@ def InjectSasAdministratorRecord(self, request):
         itself a dictionary). The dictionary is an SASAdministrator object 
         (Specified in SAS-SAS TS WINNF-16-S-0096)
     """
+    pass
+
+  @abc.abstractmethod
+  def TriggerMeasurementReportRegistration(self, request):
+    """SAS admin interface to trigger measurement report request for all subsequent
+    registration request 
+
+    Args:
+      request: A dictionary with a single key-value pair where the key is
+        "meas_report_config" and the value is a string of permitted enumerations
+        EUTRA_CARRIER_RSSI_NON_TX or EUTRA_CARRIER_RSSI_ALWAYS
```

Comment:
We might want to reference the docs instead of putting these values here -- otherwise we will have old values in the code when the set of permitted enumerations changes.